template-skeleton/template/public/favicon/browserconfig.xml

```diff
@@ -2,7 +2,7 @@
 <browserconfig>
     <msapplication>
         <tile>
-            <square150x150logo src="/favicon/mstile-150x150.png"/>
+            <square150x150logo src="/mstile-150x150.png"/>
```

Comment:
This file is placed inside the `favicon` folder, is it right?

Reply:
Correct, fixed